src/main/java/rx/Observable.java

```diff
@@ -10535,6 +10535,8 @@ public final Subscription subscribe(Subscriber<? super T> subscriber) {
     /**
      * Returns an Observable that emits those items emitted by source Observable before a specified time runs
      * out.
+     * If time runs out before Observable is finished, termination events would be called on provided {@link Scheduler},
```

Comment:
A `<p>` would cleanly separate the first sentence and the rest of the Javadoc. Also the wording could use some refinement:

```
<p>
If time runs out before the {@code Observable} completes normally, the {@code onComplete} 
event will be signaled on the default {@code computation} {@link Scheduler}.
```